### PR TITLE
docs(read-only): dropdown docs

### LINKED ISSUE #3274: [Read-only inputs]: Multi-select website docs
This docs update will be done jointly with Dropdown and Combo-box. One PR can close all three issues

### PR DESCRIPTION
**Pull request by laurenmrice**
Closes #3274, #3266, #3273 

----

**Changed**
- Updated some Usage tab image bugs
- Updated images and content on the Style tab 
- Added to updates to the Toggle and Slider pages that were missed in my previous read-only PRs

**Removed**
- Removed unneeded images

### CODE REVIEW

File: src/pages/components/dropdown/style.mdx [hunk 1]
```diff
@@ -92,13 +92,13 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 <Tab label="Default">
 
-![Default dropdown and combo-box states](images/dropdown-style-2.png)
+![Default dropdown and combobox states](images/dropdown-combobox-style-default-states.png)
```
Comment (andreancardona) [inline src/pages/components/dropdown/style.mdx:95]:
```suggestion
![Default dropdown and combo box states](images/dropdown-combobox-style-default-states.png)
```

**Review by andreancardona — APPROVED** [review]

**Review by alisonjoseph — APPROVED** [review]

**Comment by thyhmdo** [comment]
Looks good so far! Just couple things:
- Can we double check on the Multiselect if the Tag number is all disabled for Read-only? UX wise, do you think people only want to see the Text only or it's helpful to see the number of the options selected. I also checked that we don't have the Outline tag update and the Outline tag's disabled prop is not working on storybook. Other options:

image

- Related to that, I think we need to add the Tag style/token for Read-only. There's only the enable state for the tag.

**Comment by laurenmrice** [comment]
@thyhmdo The number in the tag should be $text-primary to match the "Options" text, that is a mistake in my current image! I will also add the rows for read-only tag styles 👍

**Comment by thyhmdo** [comment]
@laurenmrice Good to confirm that with you :sweat_smile: I checked in the kit and it wasn't the right one either. I'm going to update that too

**Comment by laurenmrice** [comment]
@thyhmdo Ok good to know! If this looks good, you can approve.

**Review by thyhmdo — APPROVED** [review]
looks good!

**Pull request merged**